lib/browser/api/module-keys.js

```diff
@@ -0,0 +1,48 @@
+'use strict'
+
+// TODO: Figure out a way to not duplicate this information between here and module-list
```

Comment:
it's unclear to me why this duplication is needed, could you write a little here in the comment about why it's necessary?

Reply:
Yeah I'll add a comment, high level because `module-list` require's all the browser process API files it means that `renderer/api/remote` (which now requires this file) loaded every single browser API into the renderer process bundle which broke some things.

Renderer code shouldn't depend on browser code IMO so need to figure out a way to split these concerns.  Once it's split I plan to enforce it in `filenames.auto.gni` that the renderer bundles can't depend on browser process code and vice-versa

Reply:
Added a comment